### PR TITLE
Update sample instructions

### PR DESCRIPTION
**Pull request by jeremymeng**
Some changes are ported from core-http.

Addresses #399

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
 - After cloning the repo, execute `npm install`
 
 ## Execution
 
-### node.js
-- Set the subscriptionId and token
-- Run `node samples/node-sample.js`
+### Node.js
+- Set the subscriptionId and token as instructed in `samples/node-samples.ts`
+- Run `npx ts-node samples/node-sample.js`
```
Comment (ramya-rao-a) [inline README.md:18]: npx? I don't recall using this elsewhere in our projects. Any particular wins this gives us?

Comment (xirzec) [reply README.md:18]: `npx` is just an `npm` command to execute a dependency script without installing it globally. e.g. if you install tsc as a devdep in your project you can use `npx tsc` without having to `npm install -g tsc`

Comment (jeremymeng) [reply README.md:18]: yeah it's convenient for simple commands without needing to add an entry to the package.json script section.  It even installs the package if it's not available locally.

**Comment by jeremymeng** [comment]
The CI failure indicates that we need to bump the version even for non-library changes?

**Comment by ramya-rao-a** [comment]
Since a PR merge is meant to auto publish the package, the CI fails if the version is not incremented. I can override and merge. Can you get an approval from @xirzec ?

**Review by ramya-rao-a — APPROVED** [review]

**Pull request merged**